### PR TITLE
Changed list to dictionary for Quotes

### PR DESCRIPTION
**Pull request by LazybuddyNK**
Is this fine, if not please tell.

### CODE REVIEW

File: bot/inputs.py [hunk 1]
```diff
@@ -1374,107 +1374,107 @@
...
-        "“The universe is big. It’s vast and complicated and ridiculous. And sometimes, very rarely, impossible things just happen and we call them miracles.”  ",
-        "“I am and always will be the optimist. The hoper of far-flung hopes and the dreamer of improbable dreams.” ",
-        "“The way I see it, every life is a pile of good things and bad things. The good things don’t always soften the bad things, but vice versa, the bad things don’t always spoil the good things and make them unimportant.” ",
-        "“Some people live more in twenty years than others do in eighty. It’s not the time that matters, it’s the person.”",
-        "“You know, the very powerful and the very stupid have one thing in common: they don’t alter their views to fit the facts; they alter the facts to fit their views . Which might be quite unfortunate if you happen to be one of those facts that needs altering.” ",
-        "“Never cruel or cowardly. Never give up, never give in.”",
+quo={"If you haven’t found something strange during the day, it hasn’t been much of a day.": " JOHN WHEELER",
+       "Any sufficiently advanced technology is indistinguishable from magic." : " Arthur C. Clarke",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1378]: extra whitespace around ':'
should be like this--> {"key": "value"}

File: bot/inputs.py [hunk 2]
```diff
@@ -1374,107 +1374,107 @@
...
+       "The only way of discovering the limits of the possible is to venture a little past them into the impossible."     :" Arthur C. Clarke",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1379]: extra whitespace around ':'
should be like this--> {"key": "value"}

File: bot/inputs.py [hunk 3]
```diff
@@ -1374,107 +1374,107 @@
...
+       " You cannot depend on your eyes when your imagination is out of focus."      :" Mark Twain",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1380]: extra whitespace around ':'
should be like this--> {"key": "value"}

Comment (Vyvy-vi) [reply bot/inputs.py:1380]: Extra ' ' before name--> looks weird on print.

File: bot/inputs.py [hunk 4]
```diff
@@ -1374,107 +1374,107 @@
...
+       "How wonderful that we have met with paradox. Now we have some hope for making progress."   :"  NEILS BOHR",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1381]: extra whitespace around ':'
should be like this--> {"key": "value"}

File: bot/inputs.py [hunk 5]
```diff
@@ -1374,107 +1374,107 @@
...
+        "I canna’ change the laws of physics, Captain!"       :" SCOTTY, CHIEF ENGINEER IN STAR TREK",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1382]: extra whitespace around ':'
should be like this--> {"key": "value"}

Comment (Vyvy-vi) [reply bot/inputs.py:1382]: Extra ' ' before name--> looks weird on print.

File: bot/inputs.py [hunk 6]
```diff
@@ -1374,107 +1374,107 @@
...
+        "Only those who attempt the absurd will achieve the impossible."   :" JOHN WHEELER",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1383]: extra whitespace around ':'
should be like this--> {"key": "value"}

Comment (Vyvy-vi) [reply bot/inputs.py:1383]: Extra ' ' before name--> looks weird on print.

File: bot/inputs.py [hunk 7]
```diff
@@ -1374,107 +1374,107 @@
...
+        "Someday in the next thirty years, very quietly one day we will cease to be the brightest things on Earth."         :" JAMES McAlear",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1384]: extra whitespace around ':'
should be like this--> {"key": "value"}

Comment (Vyvy-vi) [reply bot/inputs.py:1384]: Extra ' ' before name--> looks weird on print.

File: bot/inputs.py [hunk 8]
```diff
@@ -1374,107 +1374,107 @@
...
+        "- Either we are alone in the universe, or we are not. Either thought is frightening."       :" ARTHUR C. CLARKE",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1385]: extra whitespace around ':'
should be like this--> {"key": "value"}

Comment (Vyvy-vi) [reply bot/inputs.py:1385]: Extra ' ' before name--> looks weird on print.

File: bot/inputs.py [hunk 9]
```diff
@@ -1374,107 +1374,107 @@
...
+        "This foolish idea of shooting at the moon is an example of the absurd length to which vicious specialization carry scientists... the proposition appears to be basically impossible."       :" A. W. BIKERTON, 1926",
+        "The finer part of mankind will, in all likelihood, never perish-they will migrate from sun to sun as they go out.And so there is no end to life, to intellect and the perfection of humanity. Its progress is everlasting."           :" KONSTATINE E. TSIOLKOVSKY, FATHER OF ROCKETRY",
+        "The most exciting phrase to hear in science, the one that heralds new discoveries, is not “Eureka” (I found it), but 'That’s funny---."           :" ~ISAAC ASIMOV",
+        "If the man doesn’t believe as we do, we say he is a crank, and that settles it. I mean, nowadays, because now we can’t burn him."           :"   MARK TWAIN",
+        "It is impossible to travel faster than the speed of light, and certainly not desirable, as one’s hat keeps blowing off."           :"  WOODY ALLEN",
+        "'listen: there’s a hell of a good universe next door; let’s go"           :" E. E. CUMMINGS",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1391]: extra whitespace around ':'
should be like this--> {"key": "value")
Extra ' ' before name--> looks weird on print.

File: bot/inputs.py [hunk 10]
```diff
@@ -1374,107 +1374,107 @@
...
+        "There is nothing so big nor so crazy that one out of a million technological societies may not feel itself driven to do, provide it is physically possible."           :" FREEMAN DYSON",
+        "Destiny is not a matter of chance - it is a matter of choice. It is not a thing to be waited for-it is a thing to be achieved"           :" WILLIAMS JENNINGS BRYAN",
+        "IS PHYSICS INCOMPLETE?   • Answer-'Ahhhhhhhhhhhhhhhhhhhh"           :" Michio Kaku",
+        "You know you’re in love when you can’t fall asleep because reality is finally better than your dreams."           :" Dr. Suess",
+        "Get busy living or get busy dying."           :"Stephen King",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1396]: extra whitespace around ':'
should be like this--> {"key": "value")
Extra ' ' before name--> looks weird on print.

File: bot/inputs.py [hunk 11]
```diff
@@ -1374,107 +1374,107 @@
...
+        "“Big flashy things have my name written all over them. Well… not yet, give me time and a crayon."           :" ",
+        "“Never ignore coincidence. Unless, of course, you’re busy. In which case, always ignore coincidence."           :"",
+        "“Almost every species in the universe has an irrational fear of the dark. But they’re wrong. ‘Cause it’s not irrational. *I'm really scared of the dark"           :"",
+        "“Biting’s excellent. It’s like kissing – only there is a winner."           :" ",
+        ":”Everything’s got to end sometime. Otherwise nothing would ever get started."           :"",
+        "“There’s always something to look at if you open your eyes!"           :"",
+        " “Courage isn’t just a matter of not being frightened, you know. It’s being afraid and doing what you have to do anyway."           :"",
+        "There's a horror movie called Alien?That's really offensive, no wonder everyone keeps invading you..."           :"",
+        "Sometimes the only choices you have are bad ones but you still have to choose."           :"",
+        "**Never** trust a **hug**.It's just a way to hide your face..."           :""",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1463]: extra whitespace around ':'
should be like this--> {"key": "value")
Three quote marks in value field

File: bot/inputs.py [hunk 12]
```diff
@@ -1374,107 +1374,107 @@
...
+        "“This is one corner… of one country, in one continent, on one planet that’s a corner of a galaxy that’s a corner of a universe that is forever growing and shrinking and creating and destroying and never remaining the same for a single millisecond. And there is so much, so much to see."           :"",
+        "“Do you think that we care for you so little that betraying us would make a difference?"           :"",
+        "“*A BEDROOM?*,What... You've got a whole room *for not being awake in?* But what's the point? You're just missing the room!"           :"",
+        "“I've got a horrible feeling that I might have to kill you,I thought you might appreciate a drink first."           :"",
+        "“Do you know what the big problemis in telling Fantasy and Reality apart?  They're both ridiculous...."           :"",
+        "“Without the capacity for pain, we can't feel the hurt that we inflict..."           :"",
+        "I want you to know that someone cares. Someone, not me           :"",
```
Comment (Vyvy-vi) [inline bot/inputs.py:1470]: should be like this--> {"key": "value")
Missing `"` in key

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]

**Comment by Vyvy-vi** [comment]
regarding #100

**Review by Vyvy-vi — APPROVED** [review]

**Comment by Vyvy-vi** [comment]
Could have been better
I would call this work slightly untidy, but here's a suggestion: try using pylint/flake8 code linters before making a commit.
👍 
Or try: https://shorturl.at/ajsLS

**Comment by Vyvy-vi** [comment]
@all-contributors add @LazybuddyNK for docs

**Comment by Vyvy-vi** [comment]
oh wupps

**Comment by Vyvy-vi** [comment]
@all-contributors add @LazybuddyNK for code

**Comment by LazybuddyNK** [comment]
> Could have been better
> I would call this work slightly untidy, but here's a suggestion: try using pylint/flake8 code linters before making a commit.
> 👍
> Or try: https://shorturl.at/ajsLS

Thanku for suggestion, I wasn't even knowing about it.
And yes it's very helpful.

Really Thank you again, that's why I have started getting involved in open source.

**Pull request merged**